models/apartment.rb

```diff
@@ -0,0 +1,9 @@
+class Apartment < ActiveRecord::Base
+  has_many :tenants
+  attr_accessor :address, :monthly_rent, :sqft, :num_beds, :num_baths, :renters
```

Comment:
remove `attr_accessor` here and let active record define the getters and setters. wait actually maybe this is what's causing the creation error?